docs/source/TROUBLESHOOTING.rst

```diff
@@ -7,7 +7,7 @@ Troubleshooting
     :alt: View on Read the Docs
     :target: https://deepcell-kiosk.readthedocs.io/en/master/TROUBLESHOOTING.html
 
-We've done our best to make the DeepCell Kiosk robust to common use cases, however, there may be unforeseen issues. In the following (as well as on our `FAQ <http://www.deepcell.org/faq>`_, we hope to cover some possible sources of frustration. If you run across a new problem not listed in either location, please feel free to open an issue on the `DeepCell Kiosk repository <https://www.github.com/vanvalenlab/kiosk>`_.
+We've done our best to make the DeepCell Kiosk robust to common use cases, however, there may be unforeseen issues. In the following (as well as on our `FAQ <http://www.deepcell.org/faq>`_, we hope to cover some possible sources of frustration. If you run across a new problem not listed in either location, please feel free to open an issue on the `DeepCell Kiosk repository <https://www.github.com/vanvalenlab/kiosk-console>`_.
```

Comment:
```suggestion
We've done our best to make the DeepCell Kiosk robust to common use cases, however, there may be unforeseen issues. In the following (as well as on our `FAQ <http://www.deepcell.org/faq>`_, we hope to cover some possible sources of frustration. If you run across a new problem not listed in either location, please feel free to open an issue on the `DeepCell Kiosk-Console repository <https://www.github.com/vanvalenlab/kiosk-console>`_.
```
Does adding console here work? I'd like to avoid references to a general kiosk repo since we don't really have one anymore.

Reply:
I think calling it the DeepCell Kiosk-Console is getting a little more confusing. Maybe we can just put `kiosk-console` repository instead?

Reply:
Oh no! Can we code-format links with rst without it getting all weird?